frontend/src/components/sustainability/SustainabilityBoard.js

```diff
@@ -0,0 +1,142 @@
+import React, {Component} from "react";
+import DatePicker from "react-datepicker";
+import SustainabilityTable from "./SustainabilityTable";
+import {connect} from "react-redux";
+import {auth, sustainability} from "../../actions";
+import {MAX_NON_BILLABLE_TO_BILLABLE_RATIO} from "../../constants";
+
+import "react-datepicker/dist/react-datepicker.css";
+
+class SustainabilityBoard extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            startDate: this.props.sustainability.startDate,
+            endDate: this.props.sustainability.endDate,
+        };
+    }
+
+    // Because everyone counts months from 0.
+    dateString = (d) => `${d.getFullYear()}-${d.getMonth() + 1}-${d.getDate()}`;
+
+    loadAccounts = () => this.props.loadAccounts(this.dateString(this.state.startDate), this.dateString(this.state.endDate));
+
+    componentDidMount() {
+        this.loadAccounts();
+    }
+
+    prepareData(accounts, range, id) {
+        if (!accounts || !Object.keys(accounts).length) {
+            return {};
+        }
+
+        const data = {};
+
+        if (range === 'board') {
+            const cell_name = this.props.sprints.cells[id];
+            data.billable = accounts.billable_accounts.by_cell[cell_name];
+            data.non_billable = accounts.non_billable_accounts.by_cell[cell_name];
+            data.non_billable_responsible = accounts.non_billable_responsible_accounts.by_cell[cell_name];
+        } else if (range === 'user_board') {
+            data.billable = accounts.billable_accounts.by_person[id];
+            data.non_billable = accounts.non_billable_accounts.by_person[id];
+            data.non_billable_responsible = accounts.non_billable_responsible_accounts.by_person[id];
+        } else {
+            data.billable = accounts.billable_accounts.overall;
+            data.non_billable = accounts.non_billable_accounts.overall;
+            data.non_billable_responsible = accounts.non_billable_responsible_accounts.overall;
+        }
+        data.non_billable_total = data.non_billable + data.non_billable_responsible;
+        data.responsible_ratio = data.non_billable_responsible / data.billable * 100;
```

Comment:
This should be:
```suggestion
        data.responsible_ratio = data.non_billable_responsible / (data.billable + data.non_billable_responsible) * 100;
```

We want the ratio between non billable and total amount of hours to be >25%.